$NetBSD$

don't hard code perl
don't die on netbsd unknown spec

--- cgcc.orig	2014-02-16 22:44:57.000000000 +0000
+++ cgcc
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 # -----------------------------------------------------------------------------
 
 my $cc = $ENV{'REAL_CC'} || 'cc';
@@ -221,7 +222,7 @@ sub add_specs {
     } elsif ($spec eq 'linux') {
 	return &add_specs ('unix') .
 	    ' -D__linux__=1 -D__linux=1 -Dlinux=linux';
-    } elsif ($spec eq 'openbsd') {
+    } elsif (($spec eq 'openbsd') || ($spec eq 'netbsd')) {
 	return &add_specs ('unix') .
 	    ' -D__OpenBSD__=1';
     } elsif ($spec eq 'unix') {
